modules/cluster_estimation/cluster_estimation.py

```diff
@@ -81,7 +81,7 @@ def create(
         """
         Data requirement conditions for estimation model to run.
         """
-        # These parameters must be positive
+        # These parameters must be greater than or equal to 0
```

Comment:
Could you move the comments into the docstring?
```py
"""
Data requirement conditions for estimation model to run.

min_activation_threshold: Minimum number of total points before model starts running.
min_new_points_to_run: Minimum number of new points before model is run again. 0 means the model is run even if there are no new points.
max_num_components: Maximum number of clusters the model will generate.
random_state: Seed for the model. Must be >= 0 .
local_logger: Logger.
"""
```
Also reorder the checks by order of the parameters.